.pre-commit-config.yaml

```diff
@@ -121,6 +121,7 @@ repos:
   - id: trailing-whitespace
     args: [--markdown-linebreak-ext=md]
     stages: [commit]
+    exclude: ^tests/(.)*\.ambr$
```

Comment:
Would this intended to match files like `tests/.ambr`?

Reply:
> Would this intended to match files like `tests/.ambr`?

Any files with `ambr` extension under `tests` directory.